docs/usage_modes.rst

```diff
@@ -0,0 +1,147 @@
+Usage Instructions
+---------------------------------------------------------------------------------------
+
+.. _dtstt:
+
+Datasette
+^^^^^^^^^
+
+For users seeking quick, visual access to PUDL data, we have set up an experimental
+version of a web interface for PUDL tables via `Datasette <https://docs.datasette.io/en/stable/>`_,
+an open source tool that helps make data more interactive. Through Datasette, users can
+browse PUDL tables, select portions of them using dropdown menus, write SQL queries, and
+download PUDL data to CSVs. Visit our Datasette instance at
+`<https://data.catalyst.coop>`_ to begin exploring the tables. This is a work in
+progress and currently provides access to a limited amount of normalized PUDL tables.
+
+.. _zen-doc:
+
+Zenodo Archives & Docker
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Catalyst periodically publishes our fully processed data as SQLite tables and parquet
+files alongside a PUDL Docker image in a data archive on `Zenodo
+<https://zenodo.org/communities/catalyst-cooperative/?page=1&size=20>`_ . A Docker image
+functions as a set of instructions that create a "container" capable of executing a
+specified application on the Docker platform. This is useful when an application
+requires substantial setup or processing prior to use. The PUDL Docker image outlines
+the steps necessary to run the ETL so that users don't have to do it themselves every
+time they want to access the processed data or use PUDL functions.  Users can choose to
+interact directly with the processed PUDL data, through SQL or the parquet files, or run
+the Docker image to use the data within a Docker container. Using Docker provides the
+user with access to all of the denormalized output tables and analysis functions
+available through the PUDL software and is recommended for users seeking greater
+analysis capabilities.
+
+To begin using Docker, you'll need to
+`install Docker & docker-compose <https://docs.docker.com/get-docker/>`_,
+download this big compressed tarball, extract the tarball in a directory on your
+computer, and run docker image load.
+
+.. code-block:: console
+
+   $ EXAMPLE
+   $ docker image load --input filename.tar
+
+
+Make sure you don't have another Jupyter Notebook server running on port 8888 already by
+running the foreground commend ``fg``. If you do, press ``ctr c`` to quit. Continue
+testing the ``fg`` command until it outputs ``no such job``.
+
+.. code-block:: console
+
+   $ fg
+
+Run docker-compose up (need to refer to the just-loaded image, not the docker hub
+reference) Visit the URL it prints out which should start with:
+``https://127.0.0.1:8888`` or ``https://localhost:8888``
+
+.. code-block:: console
+
+    $ docker-compose up
+
+Opening the docker container will open PUDL’s JupyterLab example notebooks:
+
+* `Accessing PUDL SQLite tables <https://github.com/catalyst-cooperative/pudl-tutorials/blob/main/notebooks/01-pudl-database.ipynb>`_
```

Comment:
Bad link -- the repository has been renamed `pudl-examples`

Reply:
will let you do b/c in usage-modes module